adventuregame.rb

```diff
@@ -1,73 +1,109 @@
+# frozen_string_literal: true
+
+# == Adventure \Game
+#
+# This is gonna be such a great game! Here's some details...
+# - TODO: add details
+#
+# == Running the game
+#
+# To run the game, ... (TODO: moar details needed; etc.)
```

Comment:
Get used to documenting your code as you go. It will help you. It will also cause problems down the road, so let's talk about it. I'll provide some examples in this file...